lizard_damage/forms.py

```diff
@@ -318,6 +328,26 @@ def clean(self):
         """Checks that apply to more than one field."""
         cleaned_data = super(FormStep1, self).clean()
 
+        if not cleaned_data['customheights']:
+            # get needed AHN files
+            ahn_files = calculation._get_ahn_leaves(
+                cleaned_data['waterlevel_dataset'], logger)
+            ahn_files = [ahn[0] for ahn in ahn_files]
+            logger.debug('Required AHN %s files: %s',
+                         cleaned_data['ahn_version'],
+                         ahn_files)
+            # Check that AHN-files exists for selected AHN-version
+            ahn_data_dir = os.path.join(settings.LIZARD_DAMAGE_DATA_ROOT,
+                                        'data_ahn' + cleaned_data['ahn_version'])
+            for file in ahn_files:
```

Comment:
`file` => `ahn_file`

Ik probeer de termen altijd hetzelfde te houden ("for pietje in pietjes"). En `file` is een build-in term, dus m'n syntax highlighting krijgt er de hik van.